### ISSUE TITLE
add additional epochs

### ISSUE THREAD

**Issue by nathanielsaxe** [rstudio/keras#503]
With the example: https://keras.rstudio.com/articles/tutorial_basic_classification.html

after doing the training with `model %>% fit(train_images, train_labels, epochs = 5)`

and getting 89% accuracy, is there a way to add on more epochs without rerunning the above command but changing the epochs number?

I can't seem to figure it out or find any info.

Thanks in advance

**Comment by skeydan** [comment]
Not sure I understand your question correctly - but if you want to continue training that same model, you can just add another

```
model %>% fit(train_images, train_labels, epochs = 5) # or another number of epochs
```

**Comment by nathanielsaxe** [comment]
I think i might be misunderstanding how the command works. 

Just to clarify, if i ran 

`model %>% fit(train_images, train_labels, epochs = 5)`

then ran the same code again, would the model then have 2x5 epochs? or would it reset and just have 5 epochs?

**Comment by skeydan** [comment]
It would have trained for 10 epochs overall.
